src/weightScale/main.py

```diff
@@ -0,0 +1,132 @@
+#
+# Copyright (C) 2024-25 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import serial
+import time
+import os
+import paho.mqtt.client as mqtt
+
+# need to update code based on device used
+port_name = '/dev/ttyUSB0' 
+baud_rate = 9600
+timeout = 1  # 1 second timeout
+parity = serial.PARITY_EVEN
+byte_size = serial.SEVENBITS
+stop_bits = serial.STOPBITS_ONE
+
+MQTT_BROKER_URL = os.getenv("MQTT_URL", "127.0.0.1")
+MQTT_PORT = int(os.getenv("MQTT_PORT", "1883"))
+MQTT_TOPIC = os.getenv("MQTT_TOPIC", "event/scale")
+
+
+# Reading class object to hold data from the scale
+class Reading:
+    def __init__(self, status, value, unit):
```

Comment:
@seanohair22 you can actually put init values into the constructor here: 
```suggestion
    def __init__(self, status="", value="", unit=""):
```